### PR TITLE
Add Uncovered Types to Ecephys Tutorial

### PR DESCRIPTION
**Pull request by ehennestad**
## Motivation
Reach 100% type coverage in tutorials

## How to test the behavior?
Run ecephys tutorial

## Checklist

- [x] Have you ensured the PR description clearly describes the problem and solutions?
- [x] Have you checked to ensure that there aren't other open or previously closed [Pull Requests](https://github.com/neurodatawithoutborders/matnwb/pulls) for the same change?
- [x] If this PR fixes an issue, is the first line of the PR description `fix #XX` where `XX` is the issue number?

### CODE REVIEW

**Comment by ehennestad** [comment]
The failing test is related to this issue:
https://github.com/NeurodataWithoutBorders/nwbinspector/issues/534

File: tutorials/private/mcode/ecephys.m [hunk 1]
```diff
@@ -216,43 +259,69 @@
...
+% Add the EventDetection object to the ecephys module
+ecephys_module.nwbdatainterface.set('ThresholdEvents', event_detection);
+% Storing Spike Features (e.g Principal Components)
+% NWB also provides a way to store features of spikes, such as principal components, 
+% using the <https://neurodatawithoutborders.github.io/matnwb/doc/+types/+core/FeatureExtraction.html 
+% |*FeatureExtraction*|> class. 
+
+% Generate random feature data (time x channel x feature)
+features = rand(3, 12, 4); % 3 time points, 12 channels, 4 features
+features = permute(features, [3,2,1]) % reverse dimension order for matnwb
```
[the remark this answers is not in the document]
Comment (ehennestad) [reply tutorials/private/mcode/ecephys.m:285]: Will not work, as `'` only works for transposing 2D matrices

[the remark this answers is not in the document]
Comment (ehennestad) [reply tutorials/private/mcode/ecephys.m:285]: Could potentially just define it directly with reverse-ordered dimensions

[the remark this answers is not in the document]
Comment (bendichter) [reply tutorials/private/mcode/ecephys.m:285]: are you sure?

[the remark this answers is not in the document]
Comment (ehennestad) [reply tutorials/private/mcode/ecephys.m:285]:
```
A=rand(4,5,6);
A'
Error using  ' 
TRANSPOSE does not support N-D arrays. Use PAGETRANSPOSE/PAGECTRANSPOSE to transpose pages or PERMUTE to reorder
dimensions of N-D arrays.
```

[the remark this answers is not in the document]
Comment (bendichter) [reply tutorials/private/mcode/ecephys.m:285]: huh, ok I think I remember that working in a previous version but I might be misremembering with Python

**Pull request merged**